internal/workloadstatus/deployment.go

```diff
@@ -1,27 +1,41 @@
-package k8sutils
+package workloadstatus
 
 import (
 	"context"
-	"fmt"
+	"errors"
 	"sort"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	apiequality "k8s.io/apimachinery/pkg/api/equality"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var (
+	ErrDeploymentNotFound          = errors.New("deployment is not yet created")
```

Comment:
```suggestion
	ErrDeploymentNotFound          = errors.New("Deployment is not yet created")
```
in this case Deployment is most probably a proper noun, so it shall be capitalised

Reply:
linter complains.